app/src/main/java/com/sunasterisk/movieintroduction/data/model/ResponseGenre.kt

```diff
@@ -0,0 +1,4 @@
+package com.sunasterisk.movieintroduction.data.model
+
+class ResponseGenre( var id: Int, var name: String) {
+}
```

Comment:
format code, ưu tiên val hơn var 
Những thuộc tính nào có nhu cầu thay đổi về sau thì mới để var
nếu mặc định khởi tạo thông qua constructor thì thôi 
```suggestion
class GenreResponse(val id: Int, val name: String) {
}
```